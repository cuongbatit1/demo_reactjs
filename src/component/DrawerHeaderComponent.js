import React from "react";
import { styled, useTheme } from "@mui/material/styles";

const DrawerHeaderComponent = styled("div")(({ theme }) => ({
  display: "flex",
  alignItems: "center",
  justifyContent: "flex-end",
  padding: theme.spacing(0, 1),
  // necessary for content to be below app bar
  ...theme.mixins.toolbar,
}));

export default DrawerHeaderComponent;
