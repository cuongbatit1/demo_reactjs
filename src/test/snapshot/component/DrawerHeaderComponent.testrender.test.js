import { render, screen } from "@testing-library/react";
import DrawerHeaderComponent from "../../../component/DrawerHeaderComponent";

test("test renders DrawerHeaderComponent snapshot", () => {
  const { asFragment } = render(<DrawerHeaderComponent />);
  expect(asFragment()).toMatchSnapshot();
});
