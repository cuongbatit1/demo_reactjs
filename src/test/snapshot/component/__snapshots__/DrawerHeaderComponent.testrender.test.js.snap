// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders DrawerHeaderComponent snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1u2mxbp"
  />
</DocumentFragment>
`;
